packages/material-ui/test/integration/Select.test.js

```diff
@@ -77,4 +75,59 @@ describe('<Select> integration', () => {
       expect(getByRole('button')).to.have.text('Twenty');
     });
   });
+
+  describe('with label', () => {
+    // we're somewhat abusing "focus" here. What we're actually interested in is
+    // displaying it as "active". WAI-ARIA authoring practices do not consider the
+    // the trigger part of the widget while a native <select /> will outline the trigger
+    // as well
+    it('is displayed as focused while open', () => {
+      const { container, getByRole } = render(
+        <FormControl>
+          <InputLabel classes={{ focused: 'focused-label' }} htmlFor="age-simple">
+            Age
+          </InputLabel>
+          <Select inputProps={{ id: 'age' }} value="">
+            <MenuItem value="">none</MenuItem>
+            <MenuItem value={10}>Ten</MenuItem>
+          </Select>
+        </FormControl>,
+      );
+
+      const trigger = getByRole('button');
+      trigger.focus();
+      fireEvent.keyDown(document.activeElement, { key: 'Enter' });
+
+      expect(container.querySelector('[for="age-simple"]')).to.have.class('focused-label');
+    });
+
+    it('does not stays in an active state if an open action did not actually open', () => {
```

Comment:
This actually breaks on master confirming what we see in https://github.com/mui-org/material-ui/issues/17294